src/node_native_module_env.h

```diff
@@ -0,0 +1,63 @@
+#ifndef SRC_NODE_NATIVE_MODULE_ENV_H_
+#define SRC_NODE_NATIVE_MODULE_ENV_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "node_native_module.h"
+
+namespace node {
+class Environment;
+
+namespace native_module {
+
+class NativeModuleEnv {
```

Comment:
If this only has static methods, turn it into a namespace with functions.

Reply:
Then we'll have to make every one of these method a friend of `NativeModuleLoader`..This is primarily a proxy class for that, whose methods are all private.

Reply:
Ack.